firmware/Core/Src/telecommands/freertos_telecommand_defs.c

```diff
@@ -0,0 +1,70 @@
+
+#include "telecommands/telecommand_definitions.h"
+#include "telecommands/telecommand_args_helpers.h"
+#include "transforms/arrays.h"
+#include "unit_tests/unit_test_executor.h"
+#include "timekeeping/timekeeping.h"
+
+#include "telecommands/freertos_telecommand_defs.h"
+#include "debug_tools/debug_uart.h"
+
+
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+#include <inttypes.h>
+#include <FreeRTOS.h>
+#include <task.h>
+#include <portmacro.h>
+
+
+/// @brief Telecommand that return metadata regarding Tasks from FreeRTOS
+/// @param args_str No arguments expected
+/// @param tcmd_channel The channel on which the telecommand was received, and on which the response should be sent
+/// @param response_output_buf The buffer to write the response to
+/// @param response_output_buf_len The maximum length of the response_output_buf (its size)
+/// @return 0 if successful, >0 if an error occurred (but hello_world can't return an error)
```

Comment:
Check this comment. Please remove empty lines between `///` comment block and function definition.

Reply:
Done